apps/ledger-live-desktop/src/newArch/features/ModularDrawer/components/SelectNetwork.tsx

```diff
@@ -3,14 +3,22 @@ import { CryptoOrTokenCurrency } from "@ledgerhq/types-cryptoassets";
 import { NetworkList } from "@ledgerhq/react-ui/pre-ldls";
 import { track } from "~/renderer/analytics/segment";
 import TrackPage from "~/renderer/analytics/TrackPage";
+import { Box, Flex } from "@ledgerhq/react-ui/index";
```

Comment:
We should avoid relying on the legacy DS for this new components. Since we won't be using Tailwind in LLD directly maybe it would make sense to export a Flex component from the pre-ldls 🤔